### ISSUE TITLE
Remove --api_servers from kubelet unit file

### ISSUE THREAD

**Issue by venezia** [samsung-cnct/kraken-lib#778]
Looking at kubelet service logs:

```
Aug 22 13:18:57 ip-10-0-30-119.us-west-2.compute.internal rkt[1165]: Flag --api-servers has been deprecated, Use --kubeconfig instead. Will be removed in a future version.
```

Looking further, we provide a kubeconfig file with the API server endpoint listed, so we should be safe to remove this argument.

This is used in a bunch of files - https://github.com/samsung-cnct/k2/search?utf8=%E2%9C%93&q=--api_servers&type=

**Comment by gochist** [comment]
I think we can close this issue. It was already resolved  by #852

**Comment by coffeepac** [comment]
@gochist I agree.  closing.
